course/grader/grader.py

```diff
@@ -17,14 +17,19 @@ def grade(self, submission):
 class MultipleChoiceGrader(Grader):
 
     def grade(self, submission):
-        if submission.answer != submission.question.answer:
-            return False, 0
+        correct_count = 0
```

Comment:
A basic implementation that supports the grading of checkbox questions. Needs requirement clarifications then improvements can be made.